src/libs/location/src/character.cpp

```diff
@@ -4482,7 +4613,7 @@ void Character::UpdateAnimation()
                 case fgt_strafe_l: // Bounce to the left
                     core.Send_Message(blade, "ll", MSG_BLADE_TRACE_OFF, 0);
                     recoilSound = SOUND_INVALID_ID; // PlaySound("recoil", true);
-                    impulse += 15.0f * CVECTOR(-cosf(ay), 0.0f, sinf(ay));
+                    impulse += 10.0f * CVECTOR(-cosf(ay), 0.0f, sinf(ay));
```

Comment:
what's the reason for changing the original value? if it's required, this should be configured from scripts

Reply:
15.0 in game looks grotesque, especially when NPCs often choose dodging by strafe. Empirically 10.0 is fine.

Reply:
Agreed with @espkk, this values should be configured from scripts
If nothing sent from scripts, than vanila values should be used

Reply:
Fixed.